packages/pieces/mysql/src/lib/actions/update-rows.ts

```diff
@@ -0,0 +1,42 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { mysqlCommon, mysqlConnect } from "../common";
+
+export default createAction({
+    name: 'update_rows',
+    displayName: 'Update Rows',
+    description: 'Updates one or more rows in a table',
+    props: {
+        authentication: mysqlCommon.authentication,
+        timezone: mysqlCommon.timezone,
+        table: mysqlCommon.table(),
+        values: Property.Object({
+            displayName: 'Values',
+            description: 'Values to be updated',
+            required: true
+        }),
+        condition: Property.ShortText({
```

Comment:
This provides high flexibility, but it's hard for tech savvy, I would replace this action with `Update Row`.

This means to replace condition with `Column name` and `Column Value`, my assumption that in most cases the user will update row by id or something unique like email, If not execute query always an option.

Reply:
I thought about this as well, but it would not allow to use other operators than "=". I personally often have queries like "update everything before date x" or "disable all users that have not logged in for x days" but you are probably right, that those helpers should be as user friendly as possible. I'll change them